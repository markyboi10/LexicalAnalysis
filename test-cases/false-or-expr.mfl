2 > 3 or 2 != 2
