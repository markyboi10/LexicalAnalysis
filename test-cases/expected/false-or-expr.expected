INT(2)
GT
INT(3)
OR
INT(2)
NEQ
INT(2)
